android/build.gradle

```diff
@@ -57,7 +57,8 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
-    implementation 'org.bouncycastle:bcprov-jdk18on:1.76'
-    implementation("com.squareup.okhttp3:okhttp:4.11.0")
-    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.5.1")
+    implementation 'org.bouncycastle:bcprov-jdk18on:1.77'
```

Comment:
have you tried to build it locally + run the demo? I vaguely remember we had some problems last time we tried to bump some deps

Reply:
Yeah,  it now build locally with no errors. 

As for the demo, we'd need to do #389 and as of now, it would be much easier to test demo once we actually release the new version.